server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -79,12 +77,16 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  */
 public class JettyServerModule extends JerseyServletModule
 {
+  // This is the maximum number of threads used by jetty acceptors and selectors
```

Comment:
Given that this is a magic number based on the current implementation of Jetty that we use, please specify the breakdown of the magic number and the version of Jetty is it based on.

Reply:
added more details on computed number, also made it compute based on number of ServerConnectors being setup (we know that based on settings of whether plainText or TLS or both connectors are configured)